src/main/java/org/opentripplanner/middleware/persistence/TypedPersistence.java

```diff
@@ -122,6 +122,16 @@ public T getById (String id) {
         return mongoCollection.find(eq(id)).first();
     }
 
+    /**
+     * Retrieves the first document that matches an field value, or null if there is no match.
+     * @param field The field to scan.
+     * @param value The value to find.
+     * @return A document whose specified field has the specified value.
+     */
```

Comment:
This is the only method that has this amount of detail in the comment (@param and @return). I'm not sure what the policy is on this, but probably easier to remove to be inline with the other comments?

Reply:
Method is redundant and I went ahead and removed it.